dotnet-authserver/src/TeacherIdentity.AuthServer/Services/Csv/AzureUserImportCsvStorageService.cs

```diff
@@ -0,0 +1,26 @@
+using Azure.Storage.Blobs;
+
+namespace TeacherIdentity.AuthServer.Services.Csv;
+
+public class AzureUserImportCsvStorageService : IUserImportCsvStorageService
+{
+    private const string PendingFolderName = "pending";
+    private readonly string _connectionString;
+    private readonly string _userImportsContainerName;
+
+    public AzureUserImportCsvStorageService(IConfiguration configuration)
```

Comment:
Could we inject `BlobServiceClient` here instead of `IConfiguration`?

Reply:
Done